### ISSUE TITLE
New Feature Request: Show location in Google Maps/Earth

### ISSUE THREAD

**Issue by HunterTTP** [CCheCastaldo/mapppdr#8]
Any chance we could add the option of opening the location of a given colony in Google Maps (or Earth)? Simply have it go to the matching coordinates in a new tab of the web browser. Sometimes Google has "street view" of these regions and would add a really fun interactivity to it. 

Happy to try adding this myself if it would be merged.

**Comment by CCheCastaldo** [comment]
yes @HunterTTP, I can add this functionality, sorry for the delay.

**Comment by CCheCastaldo** [comment]
I have added this @HunterTTP. Sorry for the delay.
